### PR TITLE
Delete feature added with dialog box issue #18

### LINKED ISSUE #18: Add delete feature for emergency contacts
By clicking on the delete option , user can delete that contact from the SQLite. It should show a dialog to confirm delete before deleting the contact.

### PR DESCRIPTION
**Pull request by MayankUjawane**
•Added dialog box to confirm before deleting.

### CODE REVIEW

**Review by codeDamon — APPROVED** [review]
Good Job, bro! 
Looks perfect to me.

File: app/src/main/java/com/example/she/ViewContacts.java [hunk 1]
```diff
@@ -20,37 +23,65 @@
...
+    public void editOptionSelected(String phoneNumber) {
+        Toast.makeText(this, "Edit coming soon", Toast.LENGTH_SHORT).show();
+    }
+
+    @Override
+    public void deleteOptionSelected(final String phoneNumber) {
+
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+         builder.setTitle("Delete")
+                .setMessage("Are you sure to delete?")
```
Comment (priyalbhatewara123) [inline app/src/main/java/com/example/she/ViewContacts.java]: setMessage("Are you sure you want to delete?")
Change the message to this.

Comment (MayankUjawane) [reply app/src/main/java/com/example/she/ViewContacts.java]: Ok

Comment (MayankUjawane) [reply app/src/main/java/com/example/she/ViewContacts.java]: Done

**Comment by priyalbhatewara123** [comment]
@MayankUjawane the app crashes on deleting the contact by clicking the 3 dots.

**Comment by priyalbhatewara123** [comment]
Great work @MayankUjawane ✨

**Comment by MayankUjawane** [comment]
Thank You Ma'am

**Pull request merged**